bitwarden_license/bit-web/src/app/secrets-manager/overview/overview.component.ts

```diff
@@ -221,19 +235,27 @@ export class OverviewComponent implements OnInit, OnDestroy {
   }
 
   private locateOrganizationsWithIncomingPaymentIssues(
+    organization: Organization,
     sub: OrganizationSubscriptionResponse,
     billing: BillingResponse,
-  ) {
+  ): FreeTrial {
     const trialEndDate = sub?.subscription?.trialEndDate;
-    this.isOwner = this.organization?.isOwner;
-    this.isTrialing = sub?.subscription?.status == "trialing";
-    this.defaultPaymentSource = billing;
+    const isOwner = organization?.isOwner;
+    const isTrialing = sub?.subscription?.status == "trialing";
+    const defaultPaymentSource = billing;
     if (trialEndDate) {
       const today = new Date();
       const trialEnd = new Date(trialEndDate);
       const timeDifference = trialEnd.getTime() - today.getTime();
       this.trialRemainingDays = Math.ceil(timeDifference / (1000 * 60 * 60 * 24));
     }
+    return {
+      trialing: isTrialing,
+      remainingDays: this.trialRemainingDays,
```

Comment:
Let's remove this side effect, there should be no `this.` reference in this method, since is part of Observable
```suggestion
    const defaultPaymentSource = billing;
    let trialRemainingDays = 0;
    if (trialEndDate) {
      const today = new Date();
      const trialEnd = new Date(trialEndDate);
      const timeDifference = trialEnd.getTime() - today.getTime();
     trialRemainingDays = Math.ceil(timeDifference / (1000 * 60 * 60 * 24));
    }
    return {
      trialing: isTrialing,
      remainingDays: trialRemainingDays,
```